eslint.config.base.js

```diff
@@ -790,8 +777,8 @@ export function createConfig(opts = {}) {
           '@typescript-eslint/consistent-type-assertions': [
             'error',
             {
-              assertionStyle: 'as',
-              objectLiteralTypeAssertions: 'allow-as-parameter', // prefer `const x: T = { ... };` to `const x = { ... } as T; `
+              arrayLiteralTypeAssertions: 'allow-as-parameter', // prefer `const x: T[] = [ ... ];` to `const x = [ ... ] as T[];`
```

Comment:
New option [`arrayLiteralTypeAssertions`](https://typescript-eslint.io/rules/consistent-type-assertions/#arrayliteraltypeassertions). I'm configuring it for consistency with `objectLiteralTypeAssertions` even though ESLint was not reporting any such error. Perhaps I had to work around some when I introduced the new linting config, but it's tough to remember.